### ISSUE TITLE
Lamplighter Group

### ISSUE THREAD

**Issue by jeanbellynck** [jeanbellynck/Cayley-Graph-Generator#91]
I was asked to include the Lamplighter Group

**Comment by p4jo** [comment]
Yes! I would ask the same :D

**Comment by p4jo** [comment]
Since this group (as wreath products on infinite groups are) is not fp, we would first need some support for infinite presentations

**Comment by jeanbellynck** [comment]
Hey, My first comment for a long time!
Usually infinite presentations have a nice patterns. It is therefore possible to deduce a lot information just by looking at the first few n generators and ignoring all others. My idea for now is to have a parameter to set the generator count.

**Comment by p4jo** [comment]
Yes, this is a good idea. One thing I recently learned about is Higmans embedding theorem. I would like to try lut some examples :D (this will probably not be turned into a feature in the near future)
